pgmq-rs/src/lib.rs

```diff
@@ -440,15 +439,12 @@ impl PGMQueue {
         queue_name: &str,
         messages: &[T],
     ) -> Result<Vec<i64>, PgmqError> {
-        let mut msgs: Vec<serde_json::Value> = Vec::new();
         let mut msg_ids: Vec<i64> = Vec::new();
+        let mut q = sqlx::query(&core_query::enqueue(queue_name, messages.len(), &0)?);
```

Comment:
might have to move `core_query::enqueue(queue_name, messages.len(), &0)` into its own let statement to resolve:

```
   --> src/lib.rs:443:34
    |
443 |         let mut q = sqlx::query(&core_query::enqueue(queue_name, messages.len(), &0)?);
    |                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ - temporary value is freed at the end of this statement
    |                                  |
    |                                  creates a temporary value which is freed while still in use

```

Reply:
I pushed a change up for this. Turns out that change we merged in `pgmq-core` was breaking, so we need to get this pushed out now to resolve it. I should have caught that before, maybe we can get a test in place for it.